### PR TITLE
Added unit tests for OmiseHelper.php. Added a Github workflow to inst…

### PR DESCRIPTION
**Pull request by aashishgurung**
#### 1. Objective

Compute the code coverage automatically. Also, add Github workflow to run unit test on PR merge and setup to generate code coverage locally.

Jira Ticket: [#252](https://opn-ooo.atlassian.net/browse/SHOPIFYAPP-252)

#### 2. Description of change

Added a unit test for OmiseHelper.php class and added a Github workflow to run unit test on PR merge. A `phpunit.xml` configuration file is added. Previous test files under `Test/Model/Api` are moved under `archive`. We will improve those tests later. PR merge will also generate a new badge for code coverage that will be shown in the README.

Screen Shot 2565-07-22 at 17 12 04

**🔧 Environments:**

- Platform version: Magento 2.4.4
- Omise plugin version: Omise-Magento 2.26.0
- PHP version: 7.4.28

### CODE REVIEW

**Review by kan98 — APPROVED** [review]
LGTM nice work!

**Review by ajzkk — APPROVED** [review]

File: Helper/OmiseHelper.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
 use Omise\Payment\Model\Config\Rabbitlinepay;
 use Omise\Payment\Model\Config\Ocbcpao;
 use Omise\Payment\Model\Config\Grabpay;
-use Omise\Payment\Model\Config\Cc as Config;
```
Comment (new4762) [inline Helper/OmiseHelper.php:23]: should we still import `Cc` and update `Config` to `Cc` 🤔 
```php
    /**
     * @var array
     */
    private $cardPaymentMethods = [
        Config::CODE,
        CcGooglePay::CODE
    ];

File: Helper/OmiseHelper.php [hunk 2]
```diff
@@ -20,7 +20,7 @@
...
+use Omise\Payment\Model\Config\Config;
```
Comment (hpariwat) [inline Helper/OmiseHelper.php:23]: There's one usage that uses `Config` as `Cc`, so we may need to change it to `Cc::CODE`.

https://github.com/omise/omise-magento/blob/c3e5ab2218c60c704335aaecc79edf594b084e2e/Helper/OmiseHelper.php#L84

File: Helper/OmiseHelper.php (hunk 1 above)
Comment (hpariwat) [reply Helper/OmiseHelper.php:23]: @new4762 Nice catch!

Comment (aashishgurung) [reply Helper/OmiseHelper.php:23]: @new4762 Nice catch. Thanks. I have imported `Cc` for the code and `Config` for the config related functions.

**Review by new4762 — APPROVED** [review]

**Review by hpariwat — APPROVED** [review]

**Pull request merged**